eclipse-che/che-server
# Remove devfile validation schema mechanism

### What does this PR do?
Remove devfile validation schema mechanism and related stuff.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

fixes https://github.com/eclipse/che/issues/22755

### How to test this PR?

1. Deploy CHE from the PR image: `quay.io/eclipse/che-server:pr-640`
2. Start a workspace from a repository with the latest version of devfile, e.g. https://github.com/vinokurig/test.git

See: workspace starts successfully.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
